cmake/Install.cmake

```diff
@@ -241,8 +244,8 @@ endif()
 
 if( APPLE )
   set(CPACK_PACKAGE_DEFAULT_LOCATION "/Applications")
-  set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
-  set(CPACK_IFW_TARGET_DIRECTORY "/Applications/${CMAKE_PROJECT_NAME}-${CPACK_PACKAGE_VERSION_MAJOR}-${CPACK_PACKAGE_VERSION_MINOR}-${CPACK_PACKAGE_VERSION_PATCH}")
+  # CPACK_IFW_TARGET_DIRECTORY defaults to “@ApplicationsDir@/CPACK_PACKAGE_INSTALL_DIRECTORY”
+  #set(CPACK_IFW_TARGET_DIRECTORY "/Applications/${CMAKE_PROJECT_NAME}-${CPACK_PACKAGE_VERSION_MAJOR}-${CPACK_PACKAGE_VERSION_MINOR}-${CPACK_PACKAGE_VERSION_PATCH}")
```

Comment:
Since Windows is working, and we had to add code in here for IFW to work, I was hoping to see Linux-only changes in this PR.  It was **really** annoying to get IFW working with the CMake logic we have in here. I'm curious what this and the CPACK_PACKAGE_INSTALL_DIRECTORY line above are effectively changing.

Reply:
When doing CPack stuff, there are variables that are defined for all Cpack generators, this being one. I think it makes more sense to use these instead of harcoding generator-specific variables, because it would help with consistency, and be more future proof.

That being said in our case it's actually a change that should have no effect as best as I can tell:
* On mac: The CPACK_IFW_TARGET_DIRECTORY will be set based on this on mac, which will match the current explicit setting. (cf the block on which you commented above)
* On Windows NSIS will override it anyways (https://github.com/jmarrec/EnergyPlus/blob/develop/cmake/CMakeCPackOptions.cmake.in#L6:L9). 
* On Linux it will not be used because we use the STGZ header script. It would be used if we used the .deb package (and would be correct...)

If that makes you uneasy I can just revert it, or least harcode the `CPACK_IFW_TARGET_DIRECTORY` for mac. Let me know

Reply:
cf "Variables common to all CPack generators": specifically https://cmake.org/cmake/help/v3.2/module/CPack.html#variable:CPACK_PACKAGE_INSTALL_DIRECTORY

Reply:
Have you tested building a package on a Mac?  If not, I can try from this branch to see what happens.  As long as it works, I'm OK with the cleanup, I'm just nervous from trying to get it to work with IFW in the first place.

Reply:
I just kicked one on mac a minute ago (after needed to install QtIFW and stuff) to test it out.

Reply:
I reverted back to harcoded `CPACK_IFW_TARGET_DIRECTORY` because it would default to `“@ApplicationsDir@/CPACK_PACKAGE_INSTALL_DIRECTORY”` which ends up being `/Users/julien/Applications/EnergyPlus-9-2-0` and not `/Applications/EnergyPlus-9-2-0` like I had expected.